repos/runtime.proj

```diff
@@ -15,7 +15,7 @@
     <!-- Additional Targets -->
   <Target Name="InstallJustBuiltRuntime" AfterTargets="RemoveBuiltPackagesFromCache">
     <!-- Install the runtime that was just built to be used by downstream repos, namely, aspnetcore -->
-    <Exec Command="tar -xvf $(SourceBuiltAssetsDir)dotnet-runtime-$(runtimeOutputPackageVersion)-$(TargetRid).tar.gz -C $(DotNetCliToolDir)" />
+    <Exec Command="tar -xvf $(SourceBuiltAssetsDir)dotnet-runtime-$(runtimeOutputPackageVersion)-$(OverrideTargetRid).tar.gz -C $(DotNetCliToolDir)" />
```

Comment:
Is there any way this can break portable or non-portable builds? I don't fully understand how this was working for portable Linux builds: are those only triggered via `runtime-portable.proj`?

Reply:
This has gotten pretty convoluted... I think this works because when we do a portable build, we also always also do a non-portable build. Then on macOS (and a few others), even when built using non-portable flags, this `runtime.proj` is forced to build a non-portable RID anyway.

Reply:
@crummel @dseefeld please check me on that, I've never been too on top of the portable build.

Reply:
I think the change is correct.  It shouldn't impact non-portable builds.  In a non-portable Linux build, `TargetRid` and `OverrideTargetRid` are the same.  For OSX builds, `OverrideTargetRid` gets updated and used correctly based on this change.  For portable Linux builds, as Davis stated, both portable and non-portable builds occur.  The non-portable build runs this target via `runtime.proj` and `OverrideTargetRid` is the same as `TargetRid`.  The portable build does not run this target because it is triggered by `runtime-portable.proj`.